src/components/Calculator.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  TiDivide, TiTimes, TiMinus, TiPlus, TiEquals,
+} from 'react-icons/ti';
+
+const Calculator = () => (
```

Comment:
- [ ] Please make sure you are having class based component in calculator.js as requested in project requested. 
- [ ] i would recommend you to not use jsx. Please use js file